### PR TITLE
update translations

### PR DESCRIPTION
**Pull request by djwester**

### CODE REVIEW

File: build/lang/fr.js [hunk 1]
```diff
@@ -21,23 +21,23 @@ D2L.PolymerBehaviors.OutcomesUserProgress.LocalizeBehavior.LangFrBehavior = {
...
 		'noOutcomesStudent': 'Aucun(e) de ces {outcome, select, competencies {compétences} expectations {attentes} objectives {objectifs} outcomes {résultats} standards {normes} other {normes}} n’a été évalué(e).',
 		'noScaleInstructor': 'Pour afficher le progrès de l’apprenant vers l’atteinte de ce/cette {outcome, select, competencies {compétence} expectations {attete} objectives {objectif} outcomes {résultat} standards {norme} other {norme}}, créer et sélectionner une échelle de réalisations.',
 		'noScaleStudent': 'Aucune échelle n’a été établie pour mesurer l’atteinte des {outcome, select, competencies {compétences} expectations {attentes} objectives {objectifs} outcomes {résultats} standards {normes} other {normes}}.',
-		'noSearchResults': 'No results found for "{searchTerm}"',
+		'noSearchResults': 'Aucun résultat n\'a été trouvé pour "{searchTerm}".',
```
Comment (djwester) [inline build/lang/fr.js]: @rgoessl  just a note that I had to manually escape the single quote in this string.

Not sure if we can update the lang-build tool to handle that?

Comment (rgoessl) [reply build/lang/fr.js]: We should be able to. Make a ticket for it, I can do it at some point.

File: package.json [hunk 2]
```diff
@@ -15,23 +15,23 @@
...
     "@polymer/iron-a11y-announcer": "^3.0.2",
     "@polymer/iron-collapse": "^3.0.1",
-    "@polymer/polymer": "^3.2.0",
+    "@polymer/polymer": "^3.3.1",
     "d2l-activity-alignments": "Brightspace/d2l-activity-alignments.git#semver:^2",
-    "d2l-button": "BrightspaceUI/button#semver:^5",
-    "d2l-hypermedia-constants": "^6",
-    "d2l-icons": "BrightspaceUI/icons#semver:^6",
-    "d2l-localize-behavior": "BrightspaceUI/localize-behavior#semver:^2",
+    "d2l-button": "github:BrightspaceUI/button#semver:^5",
```
Comment (rgoessl) [inline package.json]: Were these changes intentional?

Comment (djwester) [reply package.json]: ah, no - thanks.
